nussl/separation/base/separation_base.py

```diff
@@ -120,18 +124,20 @@ def _separate(file_obj): # pragma: no cover
                 estimates.append(mix - estimates[0])
 
             estimates = {f'Estimate {i}': s for i, s in enumerate(estimates)}
-            html = play_utils.multitrack(estimates, ext='.mp3', display=False)
+            html = play_utils.multitrack(estimates, ext=ext, display=False)
```

Comment:
Maybe add the period if it's not there. `ext = f'.{ext}' if ext[0] != '.' else ext`